lib/Alchemy/Phrasea/Command/Command.php

```diff
@@ -117,11 +117,11 @@ public function getFormattedDuration($seconds)
         $duration = ceil($seconds) . ' seconds';
 
         if ($duration > 60) {
-            $duration = round($duration / 60, 1) . ' minutes';
+            $duration = round(floatval($duration) / 60, 1) . ' minutes';
```

Comment:
The original code is ugly (comparing str to int), it's better to rewrite it the clean way : 
$seconds = ceil($seconds);  // only numeric
if($seconds > 60) $duration = round(seconds/60.0, 1) . " minutes"; // duration is only str
elseif ...